### DISCUSSION TITLE
Editing @media rules in the _content directory

### DISCUSSION THREAD

**Discussion by lentrup404** [vikramlearning/blazorbootstrap#342] [Q&A]
Hey there, 

I ran into a problem while developing an Blazor Server App using Visual Studio 2022. I just started with Blazor about a month ago and ran into a problem today, that I dont know how to fix.

Basically I need to edit the @media (min-width) rules that are specified inside the css files in the _content directory (blazor.bootstrap.css and  blazor.bootstrap.bundle.scp.css). There are 2 @media rules that change the layout of the sidebar depending on the screen width of the device (Default is set to min-width: 642px, I need to change that to a bigger scale. because I need the mobile version to work on a tablet).

The problem is I can only find & edit these 2 files after I published the application. I cant find the _content folder anywhere in the /source/repos/{project} filesystem of VS22, what leads to the problem, that I cannot debug the application with the layout I need.

Now my question is, if there is any way to edit these rules without having to publish the application first. I'm fine with any workaround to get the layout I need in debug mode.

**Comment by gvreddy04** [comment]
@lentrup404 Thank you for trying **BlazorBootstrap** components. 

**Current CSS:**

```css
@media (min-width: 641px) {
    .bb-sidebar {
        width: var(--bb-sidebar-width);
        position: sticky;
        top: 0;
    }
}
```

If I see a quick work around I'll update you ASAP. Otherwise, we will do an enhancement.
